src/features/click-to-load.js

```diff
@@ -1699,48 +1707,34 @@ function createYouTubePreview (originalElement, widget) {
     return { youTubePreview, shadowRoot }
 }
 
-// Convention is that each function should be named the same as the sendMessage
-// method we are calling into eg. calling `sendMessage('getClickToLoadState')`
-// will result in a response routed to `updateHandlers.getClickToLoadState()`.
-const messageResponseHandlers = {
-    getClickToLoadState (response) {
-        devMode = response.devMode
-        isYoutubePreviewsEnabled = response.youtubePreviewsEnabled
-
-        // Mark the feature as ready, to allow placeholder replacements to
-        // start.
-        readyToDisplayPlaceholdersResolver()
-    },
-    setYoutubePreviewsEnabled (response) {
-        if (response?.messageType && typeof response?.value === 'boolean') {
-            originalWindowDispatchEvent(
-                createCustomEvent(
-                    response.messageType, { detail: response.value }
-                )
-            )
-        }
-    },
-    getYouTubeVideoDetails (response) {
-        if (response?.status && typeof response.videoURL === 'string') {
-            originalWindowDispatchEvent(
-                createCustomEvent(
-                    'ddg-ctp-youTubeVideoDetails',
-                    { detail: response }
-                )
-            )
+export default class ClickToLoad extends ContentFeature {
+    // Messaging layer between Click to Load and the Platform
+    get messaging () {
+        if (this._messaging) return this._messaging
```

Comment:
```suggestion
        if (this.#messaging) return this.#messaging
```

Reply:
If we move this to `ContentFeature` we can change this to a private property like above. (Not really a requirement at all :) )